<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserProfileDialog</class>
 <widget class="QDialog" name="UserProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>379</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="nameTitle">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Гость</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="loginLabel">
         <property name="text">
          <string>Логин: &lt;b&gt;_guest&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dateLabel">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Зарегистрирован: &lt;u&gt;год время&lt;/u&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="friendsBox">
         <property name="title">
          <string>Друзья</string>
         </property>
         <layout class="QVBoxLayout" name="friendsLayout">
          <item>
           <widget class="QPushButton" name="addByLoginButton">
            <property name="text">
             <string>Добавить друга по логину</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="noFriendsLabel">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>У вас нет друзей :(</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
